addons/web/static/src/views/kanban/kanban_arch_parser.js

```diff
@@ -104,7 +104,7 @@ export class KanbanArchParser extends XMLParser {
                 if (!widget && models[modelName][node.getAttribute("name")].type === "many2many") {
                     node.setAttribute("widget", "many2many_tags");
                 }
-                const fieldInfo = Field.parseFieldNode(node, models, modelName, "kanban", jsClass);
+                const fieldInfo = Field.parseFieldNode(node, models, modelName, "kanban", jsClass, activeActions.edit);
```

Comment:
you can have the same issue in form and list as well (if edit="0", the view will be readonly, but some widgets still allow edition in readonly, eg priority)